guide/private-key-management/single-user-schemes.md

```diff
@@ -53,9 +53,9 @@ Now let's dive in and look at the various schemes that might be suitable for a p
 
 By using an automatic cloud backup for the private key the user is NOT required to manually backup anything when setting up a new [bitcoin wallet]({{ '/guide/glossary/#bitcoin-wallet' | relative_url }}). This is sometimes also called *seedless*, as the user is never exposed to the traditional *seed*, or [recovery phrase]({{ '/guide/glossary/#recovery-phrase' | relative_url }}) that is used to initiate a wallet.
 
-This scheme can be used for those new to bitcoin, or novices who would find manually backing up a recovery phrase during onboarding confusing, possibly leading to unsafe backups. It could also be a starting point for a progressive security model that lets customers upgrade when they start accumulating more funds.
+Automatic cloud backups can be useful for a variety of user experience levels. New Bitcoin users will likely find this method more frictionless. For new users, manual backups may actually be less secure as the process is likely foreign to them. This can lead to writing down or storing the backup in an unsafe manner. More experienced users can also make use of this scheme.  Wallets that store private keys locally, also known as hot or spending wallets, likely don't need the security manual backups offer. This scheme may also bridge the gap between new and experienced users. A security model may start new users with an automatic cloud backup. Then over time shift users to more secure scheme as they accumulate more bitcoin.
 
-The automatic cloud backup scheme gives users reasonable security against theft, while providing low risk for self-inflicted losses.
+The automatic cloud backup scheme gives users reasonable security against theft whilst providing low risk for self-inflicted losses and a more seamless user experience for spending wallets of which a manual backup may be unnecessary.
```

Comment:
While appears to be the more common form.

I suggest no change to this sentence, it is repeated in a similar form for all the schemes, dealing simply with the trade-off between theft and self-inflicted loss.

Reply:
Made a commit suggestion so will resolve this comment.